x-pack/plugins/cases/public/components/all_cases/columns_popover.tsx

```diff
@@ -142,6 +200,30 @@ export const ColumnsPopover: React.FC<Props> = ({
           </EuiFlexItem>
         </EuiFlexGroup>
       </EuiDragDropContext>
+      <EuiPopoverFooter>
+        <EuiFlexGroup gutterSize="s" responsive={false} justifyContent="spaceBetween">
+          <EuiFlexItem grow={false}>
+            <EuiButtonEmpty
+              size="xs"
+              flush="left"
+              onClick={showAll}
+              data-test-subj="column-selection-popover-show-all-button"
+            >
+              {i18n.SHOW_ALL}
```

Comment:
should we hide this options when searching? It looks like you could show/hide the searched params but it actually does it for all, even for those that aren't in the list

Reply:
+1 Agree this is confusing.

Reply:
I can do that but I wrote in the PR description about it actually 😁 

> Another behavior I followed from the data grid table is clicking Show all/Hide all in the middle of a search, which will affect all columns and not just the ones displayed.

The data grid does this too @mdefazio so I left it. I will check hiding and disabling during search to see what feels better.